src/transformers/modeling_utils.py

```diff
@@ -4190,6 +4190,18 @@ def warn_if_padding_and_no_attention_mask(self, input_ids, attention_mask):
 
             logger.warning_once(warn_string)
 
+    @property
+    def _is_quantized_training_enabled(self):
+        logger.warning(
+            "`_is_quantized_training_enabled` is going to be deprecated in a future version. Please use `model.hf_quantizer.is_trainable` instead",
+            FutureWarning,
+        )
```

Comment:
Nice and safe :) Tbh, with private methods we can probably get away with no deprecation warning as it's not public.

Reply:
Yeah in my first commits I just removed that private attribute, but after giving it some thoughts I realised maybe better to go for that option just to be on the safe zone